app/models/manageiq/providers/redhat/infra_manager/event_parser.rb

```diff
@@ -78,7 +78,11 @@ def self.parse_new_vm(vm, message, event_type, ems)
   end
 
   def self.parse_target_name(message, event_type)
-    if %w(NETWORK_ADD_VM_INTERFACE NETWORK_INTERFACE_PLUGGED_INTO_VM).include?(event_type)
+    case event_type
+    when "NETWORK_ADD_VM_INTERFACE"
+      # sample message: "Interface nic1 (VirtIO) was added to VM v5. (User: admin@internal-authz)"
+      message.split(/\s/)[7][0...-1]
```

Comment:
Ok, so since this requires some work out of the scope of this PR, I'd suggest to approve this one which solves a bug around the parsing of NETWORK_ADD_VM_INTERFACE and to open a bug for replacing the logic to identify the targets based on their ID once it is possible instead.